DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/stats/stats.controller.js

```diff
@@ -0,0 +1,50 @@
+﻿(function() {
+    'use strict';
+
+    angular.module('app.stats')
+        .controller('statsController', statsController);
+    statsController.$inject = ['statsFactory', '$scope', 'logger'];
+
+    function statsController(statsFactory, $scope, logger) {
+        $scope.rep = '';
+        $scope.series = ['Series A'];
+
+        getCommits().then(function () {
+            logger.info('Got commits from GitHub!');
+        });
+
+        getRepos().then(function() {
+            logger.info('Got repositories');
+        });
+
+        function getCommits() {
+            return statsFactory.getCommitsFromRepos().then(function (response) {
+                $scope.reposData = [response.data.commits];
+                $scope.labels = response.data.months;
+                $scope.barData = [response.data.commits];
```

Comment:
Why is a bad name for variable ? I use Bar Chart that identified in ChartJs.
For example they use the same name in css
.chart-bar
chart-data: series data